counsel.el

```diff
@@ -6277,12 +6280,24 @@ The properties include:
 This variable is suitable for addition to
 `savehist-additional-variables'.")
 
+(defvar-local counsel-compile-history-match-buffer-file-name nil
+  "Make `counsel-compile-get-filtered-history' match buffer name.
```

Comment:
"match buffer name" should be "match buffer-file-name" or "match visited file name" instead, right?